src/content/docs/apm/agents/php-agent/installation/update-php-agent.mdx

```diff
@@ -24,10 +24,10 @@ To update via **tar archive**, follow the [tar archive installation procedures](
 
```

Comment:
Needs this format

title: 'Insert a title here' 
summary: 'Insert a brief description of the feature' 
releaseDate: 'yyyy-mm-dd' 
learnMoreLink (optional): 'https://example.com' 
getStartedLink (optional): 'https://example.com'